src/settings-ui/Settings.UI/ViewModels/GeneralViewModel.cs

```diff
@@ -185,6 +187,25 @@ public bool Startup
             }
         }
 
+        // Gets or sets a value indicating whether the PowerToys icon should be hidden in the system tray.
+        public bool HideSysTrayIcon
```

Comment:
Wondering... would it be better to use `ShowSysTrayIcon` rather than `Hide` so future devs don't have to think of negating in their heads.

Reply:
Understandable question, but I believe the issue is "what is the default?" and having an icon feels pretty "normal" to me. Therefore _hiding_ would be the action.

Reply:
I had chosen the negative by default for two reasons:
1. It is shown by default so this would be like "changing from the default" by setting this *flag*
2. Because icon shown is the default, any incorrect initializations will likely be 0 and not hide the icon randomly

Does anyone else have an opinion here? I'm happy to do either.